com.unity.render-pipelines.universal/Shaders/Particles/ParticlesInput.hlsl

```diff
@@ -0,0 +1,75 @@
+#ifndef UNIVERSAL_PARTICLES_INPUT_INCLUDED
+#define UNIVERSAL_PARTICLES_INPUT_INCLUDED
+
+#if PARTICLES_EDITOR
```

Comment:
Why is it that we need different attributes/varyings for editor?

Reply:
I don't know. We need to put that in our task list to look at that.

Reply:
This smells very strange :)

If i'm reading it correctly, you are saying lighting is totally disabled/broken in the Editor (because you disregard the normal + tangent inputs)

I don't think you want to ship this as-is unless it's already in this broken state in master.

Reply:
I could fix this in this PR as well but this how Editor Particles are currently on Master. I'm just moving the code into one separate file.

https://github.com/Unity-Technologies/Graphics/blob/master/com.unity.render-pipelines.universal/Shaders/Particles/ParticlesEditorPass.hlsl

Reply:
No this file you linked isn't for rendering of particles in the editor, it's for rendering the editor *specific* passes, namely the selection outline and the picking buffer version.

Reply:
It seems like I should have added a comment in this file explaining this :)
(I added it when I added instanced particle rendering support, which needs custom passes for picking+selection outlines)

Reply:
Ahh!! My bad! 
Wondering what's the best solution here. Would we be fine with moving the define into the relevant passes and keep the attributes/varyings in the same file?

Reply:
Perhaps add a define to the editor passes, then use it to exclude normal + tangents from the inputs?
You can find the passes by looking at any pass that uses the *vertParticleEditor* vertex program.

I was also thinking you could just wrap it in #if !UNITY_PASS_META but i think we use META passes for other stuff that may need those inputs. But im not sure about that. I don't know exactly what qualifies as a meta pass.